bridgesample/src/main/java/com/livefront/bridgesample/scenario/fragment/LargeDataFragment.kt

```diff
@@ -31,6 +31,23 @@ class LargeDataFragment : BridgeBaseFragment() {
             setHeaderText(R.string.large_data_header)
             generatedBitmap = savedBitmap
             onBitmapGeneratedListener = { savedBitmap = it }
+            if (getArguments(this@LargeDataFragment).infiniteBackstack) {
+                onNavigateButtonClickListener = {
+                    this@LargeDataFragment.activity!!.supportFragmentManager
+                            .beginTransaction()
+                            .replace(
+                                    (this@LargeDataFragment.view!!.parent as ViewGroup).id,
+                                    newInstance(
+                                            LargeDataArguments(
+                                                    getArguments(this@LargeDataFragment).shouldClearOnDestroy,
+                                                    true
+                                            )
+                                    )
+                            )
+                            .addToBackStack(null)
+                            .commit()
+                }
```

Comment:
Hmm, this one isn't using the `FragmentNavigationManager`

Reply:
Fixed.